.github/workflows/installer.yml

```diff
@@ -0,0 +1,75 @@
+---
+name: Installer
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'packaging/installer/install-required-packages.sh'
+  pull_request:
+    paths:
+      - 'packaging/installer/install-required-packages.sh'
+jobs:
+  install-required-packages:
+    name: Required Packages
+    strategy:
+      matrix:
+        distro:
+          - 'alpine:3.11'
+          - 'alpine:3.10'
+          - 'alpine:3.9'
+          - 'archlinux:20200106'
+          - 'archlinux:20191205'
+          - 'archlinux:20191105'
```

Comment:
This is going to be a pain to maintain because of how Archlinux works. I'd much rather test on `archlinux:latest`, as that's what actually matters for 99% of users, even if it's not entirely 'stable' as a testing platform.

Reply:
I'm fine with that.